bridges/MozillaSecurity.php

```diff
@@ -0,0 +1,26 @@
+<?php
+class MozillaSecurityBridge extends BridgeAbstract {
+
+			const MAINTAINER = 'm0le.net';
+			const NAME = 'Mozilla Security Advisories';
+			const URI = 'https://www.mozilla.org/en-US/security/advisories/';
+			const CACHE_TIMEOUT = 7200; // 2h
+			const DESCRIPTION = 'Mozilla Security Advisories';
+			const WEBROOT = 'https://www.mozilla.org/';
+
+			public function collectData(){
+					$html = getSimpleHTMLDOM(self::URI)
+							or returnServerError('Could not request MSA.');
+
```

Comment:
You can fix relative URIs using the `defaultLinkTo` function:

```suggestion

$html = defaultLinkTo($html, self::WEBROOT);

```